src/internal/opts/opts.go

```diff
@@ -0,0 +1,5 @@
+package opts
+
+type GlobalOptions struct {
+	DryRun bool
```

Comment:
I thought you declared the is dry run package variable elsewhere not in a “global” struct? I would remove this struct.